README.md

```
@@ -42,6 +42,11 @@ pnpm test -- --project node
 pnpm test -- --project node --project vue
 ```
 
+### End to End Testing
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)